docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,580 @@
+```
+
+To change the font style for standard text controls, use the following JSON syntax:
+
+```csharp
+{
+   "styles":
+   {
+      "textlabel":
+      {
+         "fontFamily": "FreeSerif",
+         "fontStyle":
+         {
+            "weight": "bold",
+            "slant": "italic"
+         },
+         "pointSize": 8
+      }
+   }
+}
+```
+
+However, the same `pointSize` is unlikely to be suitable for all text controls in an application. To define custom styles for existing controls, set a style name for each case, and provide a style override in a JSON stylesheet.
+
+You can provide further flexibility for the various screens by mapping the logical size to a physical size in the stylesheet.
+
+<a name="align"></a>
+### Align Text
+
+To align the text in a text label:
+
+-   To enable text wrapping, use the `MultiLine` property of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+
+    ```csharp
+    label.MultiLine = true;
+    ```
+
+-   To align the text horizontally to the beginning, center, or end of the available area, set the `HorizontalAlignment` property of the `Tizen.NUI.BaseComponents.TextLabel` class with the corresponding value of the [Tizen.NUI.HorizontalAlignment](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HorizontalAlignment.html) enumeration:
+
+    ```csharp
+    label.HorizontalAlignment = HorizontalAlignmentType.Begin;
+    label.HorizontalAlignment = HorizontalAlignmentType.Center;
+    label.HorizontalAlignment = HorizontalAlignmentType.End;
+    ```
+
+    The following table summarizes the available values of the `Tizen.NUI.HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script. In addition, for the illustrated examples, it is  assumed that the label size is greater than the minimum required size:
+
+    |  Alignment  | Left-to-right script example    |      Right-to-left script example |
+    |-------------|---------------------------------|-----------------------------------|
+    |  `Begin` |   ![Latin script aligned to beginning](media/LatinBegin.png) |  ![Arabic script aligned to beginning](media/ArabicBegin.png) |
+    |`Center` |  ![Latin script aligned to center](media/LatinCenter.png) | ![Arabic script aligned to center](media/ArabicCenter.png) |
+    | `End`   |  ![Latin script aligned to end](media/LatinEnd.png) |   ![Arabic script aligned to end](media/ArabicEnd.png) |
+
+
+
+<a name="decorations"></a>
+### Use Decorations
+
+For text decorations, the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class provides several properties. All the properties are editable and none of them are animatable:
+
+
+| Property    |    Type |      Description |
+|-------------|---------|------------------|
+| `Text` |  `string` |     Specifies the text to display in UTF-8 format.|

```

Comment:
Indeed, I'm not sure with the usefulness of adding "Specifies " to every sentences in the tables.

Does "A" become more clear if it is changed to "Specifies A"?
Please also consider that dummy data in the sentences can reduce the clarity.

For now, I'll keep this update as it is applied.
But for the future, @nawab-samsung, I strongly hope you to focus more on capturing out the grammar / context errors rather than changing the existing sentences, 
as there is a dangerousness of changing to wordy result.

Reply:
@mijin85cho Just to add to easy comprehension, added specifies. My intention was never to make sentences too wordy. 

If you have a better suggestion, I would be glad to go with that and suggest the developer the same.

For instance, 
If I didn't add specifies, the following description looked like this, even I have changed a lot of these to maintain parallelism in my previous comments. 
ExceedPolicy | Integer |  Specifies how the text is truncated when it does not fit.
-- | -- | --|

ExceedPolicy | Integer |  How the text is truncated when it does not fit.
-- | -- | --

Reply:
@nawab-samsung You don't need to try to change the original sentences, which means just keeping the original sentence as they are if they don't have grammar error.

You are STILL trying to change the original sentence "Specifies how the text is truncated when it does not fit." to something else. I'm telling you not to do that.

Please keep "Specifies how the text is truncated when it does not fit." and "The horizontal line alignment." as they are. We could keep below table if it doesn't have error in it. It would have  be my suggestion if the update was not applied yet.


| Property    |    Type |      Description |
|-------------|---------|------------------|
| ...    |    ... |      ... |
| ExceedPolicy |    Integer |   Specifies how the text is truncated when it does not fit.|
| HorizontalAlignment |    HorizontalAlignment |   The horizontal line alignment.|
| VerticalAlignment |    VerticalAlignment |   The vertical line alignment.|
| ...    |    ... |      ... |

There is no reason to change to "How the text is truncated when it does not fit." or "Specifies the vertical line alignment." We can keep the original.

I'm worrying that you're thinking of the consistency, not in the way I meant. It is not to change every sentences constrainedly.
Instead, please check below cases.

1. The Type column in Property table for TextLabel is using code tag unlike other two tables. Please check that Property tables for TextField and TextEditor, and ask to update Property table for TextLabel as like them.
   ![image](https://user-images.githubusercontent.com/9958374/76483291-fb58f700-6459-11ea-86af-6750e4c9380d.png)
   ![image](https://user-images.githubusercontent.com/9958374/76483318-0ca20380-645a-11ea-88f1-3e0476dfd4ce.png)
    **Property table for TextLabel**
   ![image](https://user-images.githubusercontent.com/9958374/76483335-1c214c80-645a-11ea-9742-dafeb78605d2.png)

   Also the style that Types are written is different compared to other two tables. (ex. string and String, Map and PropertyMap). Is there any reason for it? If not, please unify them.


2. Both TextField and TextEditor has `ScrollThreshold` Property, and the descriptions for them are different. Why?
   (The similar issue is happening to PrimaryCursorColor and SecondaryCursorColor.)

   **TextField Properties**

   ScrollThreshold | Float | Specifies when the horizontal scrolling will occur, if the cursor is closer to the control border.
   -- | -- | --

   **TextEditor Properties**

   ScrollThreshold | Float | Specifies whether the horizontal scrolling will occur, if the cursor is closer to the control border.
   -- | -- | --

   Even the worse point is, while adding "Specifies ...", the meaning of two sentences got different even the original for both was same "Horizontal scrolling will occur, if the cursor is closer to the control border.". So I hope you to check the sentence with @surbhig12 and suggest trimmed sentence again.

   I'm really worrying about these cases and that is the reason I keep asking you not to change the sentences and only check the issue. Please focus on issue checking rather than sentence changing.

Reply:
@mijin85cho 

The following is not original sentence, and the same was prescribed by me in earlier comments.
| Property    |    Type |      Description |
|-------------|---------|------------------|
| ...    |    ... |      ... |
| ExceedPolicy |    Integer |   **Specifies** how the text is truncated when it does not fit.|
| HorizontalAlignment |    HorizontalAlignment |   The horizontal line alignment.|
| VerticalAlignment |    VerticalAlignment |   The vertical line alignment.|
| ...    |    ... |      ... |

Saying this, the property is to be set to `ExceedPolicy` such that something will change.
- Isn't it good to make it clear to the reader that the property specifies that this action will happen rather than just writing **this happens?** or  **It means this**
-  Seems that we are giving incomplete information. 
- I am not able to understand that how does adding specifies change the meaning? rather when and whether is responsible for the same. I have not changed the context in this case.

- I always try to keep original sentences as much as I can and tweak sentences that are grammatically incorrect or contextually wrong. 

![specifies 1](https://user-images.githubusercontent.com/43839737/76490845-2c352e00-6452-11ea-9b9f-1cd95521ed8f.JPG)
![specifies2](https://user-images.githubusercontent.com/43839737/76490853-33f4d280-6452-11ea-86c4-37f4bbfa32e7.JPG)